pkg/packager/export.go

```diff
@@ -7,74 +7,61 @@ import (
 	"path/filepath"
 	"time"
 
+	"github.com/deislabs/duffle/pkg/imagestore"
+
 	"github.com/deislabs/cnab-go/bundle"
 	"github.com/docker/docker/pkg/archive"
 
 	"github.com/deislabs/duffle/pkg/loader"
```

Comment:
nit: Could you group the imports from Duffle, please?

Reply:
Done. I must admit that I hardly ever read imports blocks so I'm glad you pointed this out.